.gitignore

```diff
@@ -2,7 +2,6 @@
 /.yardoc
 /_yardoc/
 /coverage/
-/doc/
```

Comment:
`/doc/` may be ignored for a reason.  Maybe because it is the default location for `rdoc` to generate documentation?  I recommend having the example code in an `examples` directory, as that better indicates that these are executable examples.  You can still include the code in the documentation with `:include:`

Reply:
Moved to examples directory.